package.json

```
@@ -35,8 +35,9 @@
     "@trpc/next": "^10.7.0",
     "@trpc/react-query": "^10.7.0",
     "@trpc/server": "^10.7.0",
-    "next": "13.1.1",
+    "next": "^13.1.1",
```

Check if the behaviour remains the same after removing the `^`

```suggestion
    "next": "13.1.1",
```